CMakeCache.txt

```diff
@@ -0,0 +1,359 @@
+# This is the CMakeCache file.
```

Comment:
I dont think those files should be commited to repo, could you add them to gitignore?

Reply:
Sounds good! I made all the changes you requested, let me know if you need anything else!